biothings/hub/dataload/dumper.py

```diff
@@ -874,7 +874,8 @@ def set_release(self):
                 self.release = remote_dt.strftime(
                     self.__class__.RELEASE_FORMAT)
             except KeyError:
-                self.release = res.headers[self.__class__.ETAG]
+                # Use entity tag (ETag) as version number. Remove weak ETag prefix.
+                self.release = res.headers[self.__class__.ETAG].lstrip("W/")
 
```

Comment:
@ravila4 looks good to me. Just one comment for adding more comment or a link to explain why "W/" was added (is this nginx-specific?) as the context here.

Reply:
@ravila4 new comments look good. 

And I just realized a danger with `lstrip` though. `lstrip("W/")` seems to remove either `W` or `/` character, not just the leading `W/` prefix. One alternative is to check with `startswith` and then skip the first two characters by positions.

Reply:
I switched to startswith. Seems to be working with the orphandrug plugin.